### ISSUE TITLE
addon-docs: a way to override width of name column in props table

### ISSUE THREAD

**Issue by jayarjo** [storybookjs/storybook#11362]
30% for the name of the prop, especially when `addon-controls` is enabled is a bit too much:

![Screen Shot 2020-06-30 at 10 15 07 AM](https://user-images.githubusercontent.com/150075/86090133-997cd580-baba-11ea-8930-ca59ac792a00.png)

20% would have been perfect fit. Is there a way to override this somehow?

**Comment by shilman** [comment]
@domyen can you take a look at this one? i think you might've modified this in the latest round of styling updates

**Comment by jayarjo** [comment]
And I think Control column requires some `min-width` in general or it might end up with something like this:

![Screen Shot 2020-07-01 at 8 32 54 AM](https://user-images.githubusercontent.com/150075/86203204-811dc100-bb75-11ea-944e-81cb883ff02a.png)

**Comment by domyen** [comment]
@shilman got it, will take a look later this week.

**Re: column width constraints.** This speaks to a larger issue. One of the columns will always get squished when the table is narrow. For instance, in the addon panel. I wonder if we need to hide columns altogether in this case. 🤔

**Comment by jayarjo** [comment]
So there's another row that is not seen in this screenshot that contained an action with very long type and that what was stretching that column. Maybe it makes sense to put `word-break` rule on those columns too.

**Comment by jayarjo** [comment]
Maybe simply provide a way to add custom styles to components or the preview/manager pages in general. That's what I personally ended up with, but in somewhat hacky way.

**Comment by cfhull** [comment]
I have a related use case where I'm setting a long url as a default property for an avatar component.

![image](https://user-images.githubusercontent.com/5115526/89239400-0d813080-d5c7-11ea-8bc8-4ce02cad0506.png)

It looks perfect if I add  `word-break: break-all` to the cell through the dev tools.

![image](https://user-images.githubusercontent.com/5115526/89239360-f4787f80-d5c6-11ea-832d-27dc155b6b2d.png)

**Comment by domyen** [comment]
Addressed these issues here in #11805. 

If you want to customize the styles further, target these classes:
- `.docblock-argstable`
- `.docblock-argstable-head`
- `.docblock-argstable-body`

e.g., `.docblock-argstable-head thead th:first-of-type { width: 20% }`. Make sure that your selectors specificity is greater than our default styling or use `!important`.

**Comment by shilman** [comment]
Whoopee!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-rc.27 containing PR #11805 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by shilman** [comment]
Great Caesar's ghost!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.12 containing PR #11818 that references this issue. Upgrade today to try it out!

**Comment by D1no** [comment]
_(sry stumbled over this via google)_
I would like to **hide the default column** completely... is there an easy way to do this?

tldr;
I want to see the Description column only in addition to the standard none expanded controls to ensure all interfaces have comments. Too much distance between name and control for easy skimming on small and very large screens
![image](https://user-images.githubusercontent.com/2397125/157166095-d5f015ef-2ce9-4c43-9374-6fdb56c5a077.png)

**Comment by srukali** [comment]
> > hide the default column
> 
> I'm working on a Storybook 7.0 and would like to hide the Description and Default columns from the args table. Is this possible

Did anyone ever find a solution for this?
